local-volume/provisioner/pkg/discovery/discovery.go

```diff
@@ -207,12 +203,12 @@ func generatePVName(file, node, class string) string {
 	return fmt.Sprintf("local-pv-%x", h.Sum32())
 }
 
-func (d *Discoverer) createPV(file, class string, config common.MountConfig, capacityByte int64, volType string) {
+func (d *Discoverer) createPV(file, class string, config common.MountConfig, capacityByte int64, volMode v1.PersistentVolumeMode) {
 	pvName := generatePVName(file, d.Node.Name, class)
 	outsidePath := filepath.Join(config.HostDir, file)
 
 	glog.Infof("Found new volume of volumeType %q at host path %q with capacity %d, creating Local PV %q",
```

Comment:
/volumeType/volumeMode/ in this log message and throughout the codebase (e.g. L213).